### PR TITLE
📺say hello to durational stages ⏳

### PR DESCRIPTION
**Pull request by bjerkins**
I'm segmenting through different stages of rounds based on their duration defined by the data.
imbinn

Demo live at http://35.238.92.114/game/EGJWNJ

### CODE REVIEW

**Comment by bjerkins** [comment]
thanks @codecov !

**Comment by bjerkins** [comment]
slack is now connected !

File: src/components/Rounds/Quiz/Text/layout.js [hunk 1]
```diff
@@ -1,25 +1,58 @@
...
+            options: PropTypes.arrayOf(
+                PropTypes.shape({
+                    option: PropTypes.string.isRequired,
+                    correct: PropTypes.bool.isRequired,
+                }).isRequired,
+            ).isRequired,
+        }).isRequired,
+        stage: PropTypes.shape({
+            name: PropTypes.oneOf([
+                'questioning',
```
Comment (AriHrannar) [inline src/components/Rounds/Quiz/Text/layout.js]: Move to constants and make uppercase

**Review by AriHrannar — CHANGES_REQUESTED** [review]
Some small thing

**Pull request merged**